main.c

```diff
@@ -28,7 +28,18 @@
 #define ADDR_LOG_PATH "addr_log.log"
 #endif
 
+const uint8_t iv_global[16] = {164, 3,  98, 193, 52,  162, 107, 252,
```

Comment:
You should make comments around this lines.

Reply:
This is an IV for testing purpose. It may be better to give corresponding comment here